package.json

```diff
@@ -64,82 +57,27 @@
     "vsce": "^2.7.0"
   },
   "main": "./dist/extension.js",
-  "browser": "./dist/web-extension.js",
   "activationEvents": [
-    "onDebugResolve:mock",
-    "onDebugDynamicConfigurations:mock",
-    "onCommand:extension.mock-debug.getProgramName",
-    "onCommand:extension.mock-debug.runEditorContents",
-    "onCommand:extension.mock-debug.debugEditorContents"
+    "onDebugResolve:artificial-workflow",
+    "onDebugDynamicConfigurations:artificial-workflow"
   ],
   "workspaceTrust": {
     "request": "never"
   },
   "contributes": {
-    "menus": {
-      "editor/title/run": [
-        {
-          "command": "extension.mock-debug.runEditorContents",
-          "when": "resourceLangId == markdown",
-          "group": "navigation@1"
-        },
-        {
-          "command": "extension.mock-debug.debugEditorContents",
-          "when": "resourceLangId == markdown",
-          "group": "navigation@2"
-        }
-      ],
-      "commandPalette": [
-        {
-          "command": "extension.mock-debug.debugEditorContents",
-          "when": "resourceLangId == markdown"
-        },
-        {
-          "command": "extension.mock-debug.runEditorContents",
-          "when": "resourceLangId == markdown"
-        }
-      ],
-      "debug/variables/context": [
-        {
-          "command": "extension.mock-debug.toggleFormatting",
-          "when": "debugType == 'mock' && debugProtocolVariableMenuContext == 'simple'"
-        }
-      ]
-    },
-    "commands": [
-      {
-        "command": "extension.mock-debug.debugEditorContents",
-        "title": "Debug File",
-        "category": "Mock Debug",
-        "enablement": "!inDebugMode",
-        "icon": "$(debug-alt)"
-      },
-      {
-        "command": "extension.mock-debug.runEditorContents",
-        "title": "Run File",
-        "category": "Mock Debug",
-        "enablement": "!inDebugMode",
-        "icon": "$(play)"
-      },
-      {
-        "command": "extension.mock-debug.toggleFormatting",
-        "title": "Toggle between decimal and hex formatting"
-      }
-    ],
     "breakpoints": [
       {
-        "language": "markdown"
+        "language": "python"
       }
     ],
     "debuggers": [
       {
-        "type": "mock",
+        "type": "artificial-workflow",
         "languages": [
-          "markdown"
+          "python"
         ],
-        "label": "Mock Debug",
-        "program": "./out/debugAdapter.js",
-        "runtime": "node",
+        "label": "Artificial Workflow",
+        "program": "wfdebug",
```

Comment:
Note: I have not been able to get this to work; the factory in extension.ts always ignores this and creates its own debug executable with the same command name.